packages/ui/src/i18n/translations.ts

```diff
@@ -25,6 +25,7 @@ import {
   ptDict,
   zhDict,
   svDict,
+  idDict,
```

Comment:
You should add `idDict` to  `translations` variable in line 76:
https://github.com/aws-amplify/amplify-ui/blob/a9ca442f4d005a597c84a326baf18b4e9bbe78c2/packages/ui/src/i18n/translations.ts#L76-L78